Makefile

```diff
@@ -63,3 +63,6 @@ test_migration:
 
 test_gauntlet:
 	SELECTED_NETWORKS=localterra NETWORK_SETTINGS=$(shell pwd)/tests/e2e/networks.yaml ginkgo --focus=@gauntlet tests/e2e/smoke
+
```

Comment:
There is a `ginkgo` install command in the Makefile.

Should we consider rather adding `ginkgo` to our NIX & asdf envs and incentivize usage of those envs instead of adding manual installation steps?

Notice: also applies to similar Solana PR

Reply:
We could add ginkgo to asdf but there is only a plugin for the `go install` and not the three `go get` libraries we also need, so we would be separating two things that should be ran together. 

I suppose we could put together an asdf plugin that does the `go get` commands along with the `go install`. I will take a look into it to see how hard it would be because we use this same ginkgo setup in multiple repos around the company so it might be good to just push it out to a plugin.

Reply:
Yeah, try to figure out if we can make some improvements by adding it to our evns, but the current PR/setup is perfectly fine if this turns out to be complex.